# SPDX-License-Identifier: Apache-2.0
# See also: https://phabricator.wikimedia.org/T327463
# See also: https://bugs.launchpad.net/neutron/+bug/2003534
# See also: https://review.opendev.org/c/openstack/neutron/+/871274
--- /usr/lib/python3/dist-packages/neutron/agent/l3/keepalived_state_change.py	2022-09-26 16:40:29.000000000 +0000
+++ ./keepalived_state_change.py	2023-01-20 11:49:24.067756065 +0000
@@ -169,7 +169,6 @@
     config.register_common_config_options()
     config.init(sys.argv[1:])
     conf.set_override('log_dir', cfg.CONF.conf_dir)
-    conf.set_override('debug', True)
     conf.set_override('use_syslog', True)
     config.setup_logging()
     privileged.default.set_client_mode(False)
